libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -54,6 +70,20 @@ export class ReadingListEffects implements OnInitEffects {
     )
   );
 
+  undoRemoveBook$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ReadingListActions.addToReadingList),
```

Comment:
Are you sure about this? Shouldn't this be handling your undo action?

Reply:
yes, changing to use the correct action here and also line 43